cfme/common/__init__.py

```diff
@@ -26,6 +26,22 @@ class ManagePoliciesView(BaseLoggedInPage):
     def is_displayed(self):
         return False
 
+    def check_policy_assign_unassign(self, assign, *policy_profile_names):
+        """ check if at list one of the given policy profiles,
+        need to be assign or unassign.
+
+        Args:
+            assign: Wheter to assign or unassign policy.
+            policy_profile_names: :py:class:`str` with Policy Profile names.
+        """
+
+        for policy_profile in policy_profile_names:
+            leaf = self.policy_profiles.expand_path(policy_profile)
```

Comment:
**Required**

Use `CheckableBootstrapTreeview.node_checked` instead of manually getting the leaf element.

Reply:
Remove method check_policy_assign_unassign, since i used CheckableBootstrapTreeview directly within its fill() method